tests/core/test_integration/test_train.py

```diff
@@ -35,5 +35,5 @@ def test_train(tmpdir):
     sys.argv = train_args
     train_run()
 
-    model_file = os.path.join(tmpdir, "model.h5")
+    model_file = os.path.join(tmpdir, "model.keras")
```

Comment:
is this the new default extension? This should be added to https://github.com/brainglobe/brainglobe.github.io/pull/189 (I think there's a couple of places in the docs that reference the .h5 files directly).

Reply:
If we're saving the whole model then we have to use the `.keras` extension. For weights, the files must now end with `.weights.h5`. I'll double check for references to `.h5` in our documentation.